sdk-setup/src/sdk-assistant

```diff
@@ -36,6 +37,7 @@ set -o nounset
 
 fail() {
     echo "FAIL: $@"
```

Comment:
something like:
```
echo "unknown option: $@"
```
Sounds less fatal but I can live with these as well.

Reply:
If we remove the prefix, then the messages about missing argument to --user and --pasword option should be changed to not start with the option name.

Reply:
> If we remove the prefix, then the messages about missing argument to --user and --pasword option should be changed to not start with the option name.

Any suggestions for better error messages in these cases? Or should I just prepend "FAIL: " to those?

Reply:
After trying this out, I came to conclusion that not having the prefix feels a bit weird too. Maybe just "error:" instead of "FAIL:" - it would not have quite as fatal feeling, but still a bit of "something went wrong" feeling is present.

Reply:
"sdk-assistant: " would work well for me as the prefix.

Reply:
Same here.

Reply:
The obvious choice :)

Reply:
Replaced the prefix with "sdk-assistant: ". I also changed the prefix in the case when no command is given to match.